anger.js

```diff
@@ -18,24 +18,28 @@ function anger (opts) {
   const latencies = new Histogram(1, 10000, 5)
   const identifier = opts.identifier || 'id'
   const tail = opts.tail || false
-  const auth = opts.auth
   const expectedResponses = typeof opts.responses === 'number'
     ? opts.responses
     : opts.connections * opts.requests
   const map = new Map()
   const uidOf = typeof identifier === 'function'
     ? identifier
     : (payload) => get(payload, identifier)
+  let auth = _noop
+  if (opts.auth) auth = typeof opts.auth === 'function' ? opts.auth : function () { return opts.auth }
 
-  for (var i = 0; i < clients.length; i++) {
+  for (let i = 0; i < clients.length; i++) {
     clients[i] = new Client(opts.url)
+    clients[i]._angerIndex = i
+    clients[i].sender = false
     if (i < opts.senders) {
       senders[i] = clients[i]
+      clients[i].sender = true
```

Comment:
these should be consistent. I think we can add a property `anger` and attach things there, e.g.

```js
clients[i].anger = {
  id: i,
  sender: true
}
```